### ISSUE TITLE
os_family choices in operatingsystem appears to be outdated

### ISSUE THREAD

**Issue by lpther** [theforeman/foreman-ansible-modules#1435]
##### SUMMARY

When using the role theforeman.foreman.operatingsystem, I want to set the operating system "Fedora CoreOS" but I am unable to set it from this collection.

##### ISSUE TYPE
 - Bug Report

##### ANSIBLE VERSION

```
ansible [core 2.12.5]
  config file = /data/home/lpt069/workspace/svc_baseos_provisioning/ansible.cfg
  configured module search path = ['/data/home/lpt069/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /data/home/lpt069/.local/lib/python3.8/site-packages/ansible
  ansible collection location = /data/home/lpt069/.ansible/collections/ansible_collections:/data/home/lpt069/workspace/svc_baseos_provisioning_includes/ansible_collections:/data/home/lpt069/workspace/svc_baseos_provisioning
  executable location = /data/home/lpt069/.local/bin/ansible
  python version = 3.8.10 (default, Mar 15 2022, 12:22:08) [GCC 9.4.0]
  jinja version = 3.1.1
  libyaml = True
```

##### COLLECTION VERSION

```
Collection            Version
--------------------- -------
ansible.posix         1.3.0  
community.general     5.0.0  
community.vmware      2.6.0  
theforeman.foreman    3.4.0  
theforeman.operations 1.2.0 
```

##### KATELLO/FOREMAN VERSION

```
package tfm-rubygem-katello is not installed
foreman-3.2.1-1.el8.noarch

```

##### STEPS TO REPRODUCE

```yaml
  theforeman.foreman.operatingsystem:
    username: "{{ foreman_username }}"
    password: "{{ foreman_password }}"
    server_url: "{{foreman_url }}"
    validate_certs: "{{ foreman_validate_certs }}"
    description: Fedora CoreOS 36
    name: FedoraCoreOS
    major: 36
    minor: 20220605.3.0
    os_family: Fedora CoreOS
    password_hash: SHA256
    architectures: [ x86_64 ]

fatal: [localhost]: FAILED! => changed=false 
  msg: 'value of os_family must be one of: AIX, Altlinux, Archlinux, Coreos, Debian, Freebsd, Gentoo, Junos, NXOS, Rancheros, Redhat, Solaris, Suse, Windows, Xenserver, got: Fedora CoreOS'
```

##### EXPECTED RESULTS

To be able to select the operating system Fedora CoreOS as when editing via the web interface.

It might be the OS_LIST in plugins/module_utils/foreman_helper.py that is outdated, but might knowledge in proposing an actual fix with proper tests is lacking.

##### ACTUAL RESULTS
```
fatal: [localhost]: FAILED! => changed=false 
  msg: 'value of os_family must be one of: AIX, Altlinux, Archlinux, Coreos, Debian, Freebsd, Gentoo, Junos, NXOS, Rancheros, Redhat, Solaris, Suse, Windows, Xenserver, got: Fedora CoreOS'
```

**Comment by evgeni** [comment]
Per https://github.com/theforeman/foreman/blob/877eb770fb20b3bdb541114ad671d6a74ed4dd9f/app/models/operatingsystem.rb#L61 the family ia Fcos and it seems to be missing in our list indeed!

**Comment by evgeni** [comment]
This was fixed in #1448 and released in 3.5.0
